common-healing.lic

```diff
@@ -138,17 +138,24 @@ module DRCH
       /^[\w]+ (?:body|skin) is covered (?:in|with) open oozing sores/
     ])
 
+    dead_regex = Regexp.union([
+      /^(He|She) is dead/
+    ])
+
```

Comment:
What is the purpose of `Regexp.union` here, given you're only specifying one regex pattern? `.union` will give a union of multiple patterns.

Reply:
In an earlier draft I had the two lines separate and followed the existing convention. I don't think we'll ever had to add a line here though. I'll fix in a bit.

Reply:
Actually, I rather keep it like this. It's all the same for the result, but this syntax leaves open simple additions for this regex (even if I can't think of any today) and is simple to read.